src/sas/qtgui/Plotting/Slicers/BaseInteractor.py

```diff
@@ -119,16 +118,29 @@ def onLeave(self, ev):
     def onClick(self, ev):
         """
         Prepare to move the artist.  Calls save() to preserve the state for
-        later restore().
+        later restore(). Also notify plotter of slicer interaction.
         """
         self.clickx, self.clicky = ev.xdata, ev.ydata
         self.save(ev)
+        try:
+            # Case 1: this interactor is the slicer (base is the plotter)
+            if hasattr(self.base, 'notifySlicerModified'):
+                self.base.notifySlicerModified(self)
+            # Case 2: this interactor is a child of the slicer (base is the slicer)
+            elif hasattr(self.base, 'base') and hasattr(self.base.base, 'notifySlicerModified'):
+                self.base.base.notifySlicerModified(self.base)
+        except Exception:
```

Comment:
This is dangerous! What kind of exceptions are we expecting? If they are benign, let's catch them with proper signatures and add comments on why we drop them.